Sources/Kitura/contentType/ContentType.swift

```diff
@@ -19,8 +19,16 @@ import LoggerAPI
 
 // MARK: ContentType
 
-/// A set of APIs to work with Content-Type headers, whether to generate the value
-/// or to determine if it's an acceptable value.
+/**
+ The `ContentType` class provides functions to determine the MIME content type for a given file extension. The user can pass in a complete file name e.g. "foo.png" or just the file extension e.g. "png", or they can pass in both a MIME content type and a file extension and query whether they match.
+ ### Usage Example: ###
+ In this example, a `ContentType` instance is initialised called contentType. This instance is then used to obtain the MIME content type of the file "foo.png", which is identified as "image/png".
+ ```swift
+ let contentType = ContentType.sharedInstance
+ let result = contentType.getContentType(forFileName: "foo.png")
+ print(String(describing: result)) // "image/png"
+ ```
```

Comment:
Please can we make this look more like Apple's Swift documentation by moving the "image/png" to the next line with the word Prints in front - ie
// Prints Optional("image/png")